src/operator/quantization/quantized_pooling.cc

```diff
@@ -78,19 +81,39 @@ bool QuantizedPoolingType(const nnvm::NodeAttrs& attrs,
   const PoolingParam& param = nnvm::get<PoolingParam>(attrs.parsed);
   CHECK_EQ(in_type->size(), 3U);
   CHECK_EQ(out_type->size(), 3U);
-  if (param.pool_type == pool_enum::kMaxPooling || param.pool_type == pool_enum::kAvgPooling) {
```

Comment:
Why remove this?

Reply:
Will add MXNET_USE_MKLDNN check and only bypass the check for MKLDNN.